test/infoSpec.js

```diff
@@ -7,17 +7,17 @@ const ObjStore = require('./helpers/objStore')
 const PluginVirtual = require('..')
 
 const info = {
+  currencyScale: 2,
   currencyCode: 'USD',
-  currencySymbol: '$',
   precision: 15,
-  scale: 15,
```

Comment:
How come scale is removed here, but not in the other test suites? Is it being kept around for backwards compatibility, or removed?